README_specifications.md

```diff
@@ -0,0 +1,55 @@
+chitter-challenge specifications:
+
+```
```

Comment:
The README should ideally have technologies used, how to run the webserver and screen shots of your app